### ISSUE TITLE
ColorPickerHueRing's Slider doesn't work if current color is #FFFFFFFF or #FF000000

### ISSUE THREAD

**Issue by glaceon2000** [mchome/flutter_colorpicker#79]
Can we make it that the slider would work regardless of the current chosen color?
https://user-images.githubusercontent.com/51567674/171573972-3d8f4d45-d4a1-44ec-87ef-6960a4cd3a6f.mov

**Comment by app97204** [comment]
Yes, me too. But mine is just black and white, there aren't any other color in the left of picker.

**Comment by chiuhol** [comment]
Has the problem been solved？

**Comment by santiihoyos** [comment]
same problem

**Comment by DanielAndresClavijo** [comment]
I also have this problem, still no solution?

**Comment by cledsonv** [comment]
same problem
